### DISCUSSION TITLE
Haproxy error

### DISCUSSION THREAD

**Discussion by acore-point** [autobase-tech/autobase#266] [Q&A]
pgnode01 haproxy[2859147]: [WARNING]  (2859147) : Server replicas_async/pgnode02 is DOWN, reason: Layer7 wrong status, code: 503, info: "Service Unavailable", check duration: 1ms. 1 active and 0 backup servers left. 0 ses>

pgnode1 cannot see pgnode2 and pgnode3

**Comment by vitabaks** [comment]
Please make sure that the [ports](https://github.com/vitabaks/postgresql_cluster#port-requirements) are accessible between the servers.
